packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -981,6 +982,15 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.mediaConnections = null;
 
+    /**
+     * @instance
+     * @type {Object}
+     * @readonly
+     * @public
+     * @memberof Meeting
+     */
+    this.diagnosticsInfo = {};
```

Comment:
could also store this on statsAnalyzer itself instead of here, thoughts?

Reply:
Makes more sense on statsAnalyzer, and one less event to listen to here.

Reply:
yes, definitely all this should be done through existing statsAnalyzer

Reply:
done ! I'm cool with that